src/main/webapp/WEB-INF/jsp/managestudy/viewStudySubject.jsp

```diff
@@ -1767,6 +1776,19 @@
             jQuery.blockUI({message: jQuery('#editSubjectForm'), css: {left: "300px", top: "10px"}});
         });
 
+        jQuery('#btn-copy button').click(function() {
+          var accessCode = document.getElementById('access-code-input');
+          accessCode.select();
+          accessCode.setSelectionRange(0, 99999); // for mobile devices
+          var copied = document.execCommand("copy");
+          if (copied) {
+            alert('<fmt:message key="copy_to_clipboard_success" bundle="${resword}"/>');            
```

Comment:
I don't think we want to show alert messages to the user when the copy succeeds or fails. This would require an additional click for the user.
Instead, can we show a message below the input box that the access code is copied?

Reply:
@svadla-oc OK done in 3c9f5566f168c0db3114c843d0f0a8974f806fd4